js_modules/dagster-ui/packages/ui-core/src/app/time/Timestamp.tsx

```diff
@@ -7,14 +7,19 @@ import {timestampToString} from './timestampToString';
 interface Props {
   timestamp: {ms: number} | {unix: number};
   timeFormat?: TimeFormat;
+  dateTimeSeparator?: string;
```

Comment:
Can we do these date-time changes in a separate PR?

I'm skeptical about making one-off changes for this specific timestamp display, especially if it involves modifying natively formatted strings. E.g. "at" might not make any sense to be inserted in a non-en-US locale, and there's no reason not to be consistent about how we show timestamps throughout the app.

Reply:
yeah I guess, I was trying to do what my figma overlord wanted